scripts/build_renewable_profiles.py

```diff
@@ -406,6 +411,28 @@ def get_hydro_capacity_annual_hydro_generation(
         if "natura" in config and config["natura"]:
             excluder.add_raster(paths.natura, nodata=0, allow_no_overlap=True)
 
+            natura = rio.open(paths.natura)
+            if not natura.crs == area_crs:
+                logger.warning(
+                    f"Coorginate referense system of 'natura.tiff' raster is {natura.crs} which is different from area_crs == {area_crs}"
+                )
+
+            # Spatial extent of the natura.tiff raster should cover the entire cutout area to avoid data losses
+            natura_orig_geom = loads(box(*natura.bounds).wkt)
+            natura_crs = pyproj.CRS(natura.crs)
+            project = pyproj.Transformer.from_crs(
```

Comment:
Is there a reason why you used pyproj for reprojecting instead of rasterio?
If possible, I'd go for rasterio to avoid adding new packages to the repo

Reply:
Agree. Will try to replace with rasterio

Reply:
Fixed. With rasterio it looks even better :)